File: C++/프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list) {
    vector<int> answer;
    
    int N = num_list.size();
    int num1 = num_list[N-1];
    int num2 = num_list[N-2];
    
    int t;
    
    if (num1 > num2) {
        t = num1 - num2;
    } else {
        t = num1 * 2;
    }
    
    num_list.push_back(t);
    
    return num_list;
}

File: JavaScript/프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.js
function solution(num_list) {
    let answer = [];
    const N = num_list.length;
    const n1 = num_list[N-1];
    const n2 = num_list[N-2];
    let a = 0
    if (num_list[N-1] > num_list[N-2]){
        a = n1 - n2;
    } else {
        a = n1 * 2;
    }
    
    num_list.push(a);
    answer = num_list;
    
    return answer;
}